src/services/ui/src/components/ExportButton/index.tsx

```diff
@@ -35,6 +37,16 @@ export const ExportButton = <TData extends Record<string, any>>({
     for (const item of resolvedData) {
       const column: Record<any, any> = {};
 
+      const checker = PackageCheck(item);
+      if (checker.hasEnabledRaiWithdraw) {
+        item.stateStatus =
+          item.stateStatus +
+          ` (${stateUserSubStatus.WITHDRAW_FORMAL_RAI_RESPONSE_ENABLED})`;
+        item.cmsStatus =
+          item.cmsStatus +
+          ` (${stateUserSubStatus.WITHDRAW_FORMAL_RAI_RESPONSE_ENABLED})`;
+      }
+
```

Comment:
Bespoke injections in exports leads to dependencies far from where data is aggregated and structured. I will reiterate what I originally said with this work: I believe substatus should be an object property rather than ad-hoc included in UIs and exports.